Remotion/ObjectBinding/Web.Test/IndividualControlTests/Form.aspx

```diff
@@ -44,7 +44,26 @@
       </View>
       
       <BottomControls>
-         <asp:UpdatePanel ID="StackUpdatePanel" runat="server" UpdateMode="Conditional">
+        <asp:CheckBox ID="MainContentScrollableCheckBox" runat="server" AutoPostBack="True" Text="Main content scrollable" OnCheckedChanged="MainContentScrollableCheckBox_OnCheckedChanged"/>
```

Comment:
I'd like to do a pair-review for the ASPX stuff to see if there're sensible improvement options.